RecFCCeeCalorimeter/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
 # ONNX DEPENDENCIES
 # includes
-include_directories("${ONNXRUNTIME_INCLUDE_DIRS}")
+include_directories(${ONNXRUNTIME_INCLUDE_DIRS})
```

Comment:
Is this intended?

Reply:
No, I'll revert this -- this was for some local changes that I was testing.